File: JS_version/functions/continuation.js
import { parse } from "path";
import readline from "readline";

const rl = readline.createInterface({
  input: process.stdin,
  output: process.stdout,
});

export function continuation(netChip) {
  return new Promise((resolve) => {
    const askQuestion = () => {
      rl.question("Do you want to continue??? (Y/N): ", (playerCommand) => {
        if (playerCommand == "n" || playerCommand == "no") {
          rl.close();
          console.log(`You got total ${netChip} chips`);
          resolve(true);
        } else if (playerCommand == "y" || playerCommand == "yes") {
          rl.close();
          resolve(false);
        } else {
          console.log("Please enter the correct command...");
          askQuestion();
        }
      });
    };

    askQuestion();
  });
}

File: functions/continuation.py
def continuation( net_chip):
    exit_command = False
    while True:
        player_command = input("Do you want to continue???(Y/N): ")
        if player_command.lower() == "n" or player_command.lower() == "no":    
            print(f"You got total {net_chip} chips")
            exit_command = True
            break
        elif player_command.lower() == "y" or player_command.lower() == "yes":
            break
        else:
            print("Please enter the correct command...")
    return exit_command